tests/data/api/api-summary-table-set-data.json

```diff
@@ -1,47 +1,4 @@
 {
   "data": [],
```

Comment:
I believe the test that generates this was written before the addition of `published_only`; I changed the test to use `published_only=False` so that actual data comes back to this record.